Update securing_applications guide for latest adapter changes (community) (#20995)
keycloak/keycloak: docs/documentation/securing_apps/topics/saml/java/jetty-adapter.adoc

==== Jetty SAML adapters

> WARNING: {project_name} Jetty SAML adapter is deprecated. It is recommended to use other client adapter if possible.

Reviewer: ```suggestion
WARNING: The {project_name} Jetty SAML adapter is deprecated. We recommend that you use another client adapter if possible.
```
